lib/bin/rebuild_phusion_passenger_image

```diff
@@ -14,22 +14,27 @@ echo '########################################'
 
 PHUSION_PASSENGER_IMAGE_NAME="$(new_image_name_from_repo $PHUSION_PASSENGER_IMAGE_REPO)"
 IMAGE_FULLNAME="$DOCKER_NAMESPACE/$PHUSION_PASSENGER_IMAGE_NAME"
-# add -full to target image to match Dockerfile
-TARGET_IMAGE_NAME="$IMAGE_FULLNAME-full:$PHUSION_PASSENGER_IMAGE_VERSION"
+# add -ruby32 to target image to match Dockerfile
+TARGET_IMAGE_NAME="$IMAGE_FULLNAME-ruby32:$PHUSION_PASSENGER_IMAGE_VERSION"
 
 github_checkout "$PHUSION_PASSENGER_IMAGE_REPO" "$PHUSION_PASSENGER_IMAGE_SOURCE_DIR" "$PHUSION_PASSENGER_IMAGE_REPO_RELEASE_TAG" || exit 1
 
 cd "$PHUSION_PASSENGER_IMAGE_SOURCE_DIR" || exit 1
 
 export PATH="$BASE_DIR/lib/docker_shim:$PATH" # use a docker shim to filter out arguments we don't like
 
+# remove registry prepend from Makefile
+MAKEFILE="Makefile"
+sed -i "s/NAME = \$(REGISTRY)\//NAME = /" $MAKEFILE
+# fix Dockerfile.base and Dockerfile use correct images
+BASE_DOCKER_FILE="./image/Dockerfile.base"
+sed -i "s~^FROM[[:space:]].\{1,\}$~FROM $DOCKER_NAMESPACE/phusion_baseimage:$PHUSION_BASE_IMAGE_VERSION~" $BASE_DOCKER_FILE
 DOCKER_FILE="./image/Dockerfile"
-sed -i "s~^FROM[[:space:]].\{1,\}$~FROM $DOCKER_NAMESPACE/phusion_baseimage:$PHUSION_BASE_IMAGE_VERSION~" $DOCKER_FILE
-
-MAKE_TARGET=build_full
+sed -i "s~^FROM[[:space:]].\{1,\}$~FROM $DOCKER_NAMESPACE/phusion_passenger-base\:current-\$ARCH~" $DOCKER_FILE
+MAKE_TARGET=build_ruby32
 MAKE_OUTPUT="$(make $MAKE_TARGET NAME="$IMAGE_FULLNAME")" || exit_with_error_message "make build FAILED" # build
 # due to mismatch of version tag in Makefile vs. version.txt, we must manually tag built image with correct name
-# since build_full will create an arm64 and amd64 image, we need to grab the correct (amd64) image id
+# since build_ruby33 will create an arm64 and amd64 image, we need to grab the correct (amd64) image id
```

Comment:
Is it worth saving the arm64 image in case some odd behavior in development on a Mac comes from AMD vs ARM differences?

Reply:
Good question, but I would say no - this automation is used 99% of the time for deployed environments, and the `amd64` is the one we want there.  Since we've abandoned local development in Docker I don't think there's a big need to keep the `arm64` image.